pyvista/plotting/tools.py

```diff
@@ -473,7 +487,15 @@ def opacity_transfer_function(mapping, n_colors, interpolate=True, kind='quadrat
         try:
             return transfer_func[mapping]
         except KeyError:
-            raise KeyError(f'opactiy transfer function ({mapping}) unknown.')
+            tf_str = list(transfer_func.keys())
+            tf_str.sort()
+            allowable = (''.join([f'    * "{item}"\n' for item in tf_str]))[:-1]
+            msg = (
+                f'Invalid opactiy transfer function "{mapping}". Should be one '
+                f'the following:\n\n{allowable}'
+            )
+            raise KeyError(KeyErrorMessage(msg))
```

Comment:
Just a drive-by remark for now: `KeyErrorMessage` is pretty hacky (I assume the "bug" is still there because you rarely have newlines in keys, and when you do you would _want_ the repr of the key to be printed).

Since it's an implementation detail that we have a dict, how about we just raise a `ValueError`? With my usual recommendation to do `raise ... from None`:
```suggestion
                f'Invalid opacity transfer function "{mapping}". Should be one of '
                f'the following:\n\n{allowable}'
            )
            raise ValueError(msg) from None
```